scripts/build-vercel-dist.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+LOG_FILENAME="build-output.log"
+ERROR_STRING="error"
```

Comment:
@VasylMarchuk would refrain from indirection unless we're going to re-use a variable, or it's a very "configurable" kind of value. In this case I'd just inline the value, extracting it out doesn't make it any easier to understand what's happening here.

Extracting `LOG_FILENAME` is fine since it's re-used multiple times.